### ISSUE TITLE
UI spring cleaning: tags list

### ISSUE THREAD

**Issue by codebykat** [Automattic/simplenote-electron#2566]
https://app.zeplin.io/project/5d795a3d8ed5261aa146e057/screen/5fbbc5352163646a2d1b37ea
https://app.zeplin.io/project/5d795a3d8ed5261aa146e057/screen/5fbbc5944b88d656997f593a

* Re-style section headers
* Remove blue / verify color values
* background color in dark mode
* Tag list should now appear on top of the content rather than pushing everything over

<img width="257" alt="Screen Shot 2021-01-10 at 10 51 49 PM" src="https://user-images.githubusercontent.com/52152/104153177-84043480-5396-11eb-964d-b008948181e1.png">

<img width="357" alt="Screen Shot 2021-01-10 at 10 53 54 PM" src="https://user-images.githubusercontent.com/52152/104153256-bc0b7780-5396-11eb-825c-66e93917d507.png">

**Comment by FedeCuci** [comment]
+1 for me. I would instead opt to include a 'Tags' menu in the original view of the app instead of in the options menu. I have included a picture of what I mean. Sorry for my terrible html/css skills. Basically allow the user to browse tags just as he/she would browse notes, using the 2 buttons I have added in the picture. 
![Picture1](https://user-images.githubusercontent.com/19730248/104947299-da3b2e00-59bb-11eb-89ec-0fb8e8f464f5.png)
